src/accordion/accordion.ts

```diff
@@ -45,27 +17,48 @@ export class NgbAccordion {
     </div>
   `
 })
-export class NgbAccordionGroup implements OnDestroy {
-  private isDisabled: boolean;
-  private _isOpen: boolean = false;
+export class NgbAccordionGroup {
```

Comment:
`NgbAccordionGroup` name sounds off. Bootstrap is using "Collapsible Group _Item_" as a name. My suggestion is to stick to `NgbPanel` and make a panel an independent component (not tied to accordion).